pkg/nodeconfig/nodeconfig_test.go

```diff
@@ -49,3 +49,63 @@ func Test_getClusterAddr(t *testing.T) {
 		})
 	}
 }
+
+func TestGetClusterDNSIP(t *testing.T) {
+	type args struct {
+		iprange string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+			name:    "empty iprange",
+			args:    args{iprange: ""},
+			want:    "",
+			wantErr: true,
```

Comment:
Shouldn't `:` all line up? Is your formatter configured correctly?

Reply:
Not following you. Currently using default `gofmt` and this is the result.

How different is the style from [line#24](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/nodeconfig/nodeconfig_test.go#L24)?

Reply:
OK, I was mistaken about this.